api/openapi/api.yaml

```diff
@@ -676,5 +676,42 @@ paths:
                     ]
         "500":
           description: Check schematic log. 
+      tags:
+        - Schema Operation
+
+  /explorer/get_node_ranges:
+    get:
+      summary: Get all the valid values that are associated with a node label.
+      description: Get all the valid values that are associated with a node label.
+      operationId: api.routes.get_node_ranges
+      parameters:
+        - in: query
+          name: schema_url
+          schema:
+            type: string
+          description: Data Model URL
+          example: >-
+            https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld
+          required: true
+        - in: query
+          name: node_label
+          schema:
+            type: string
+            nullable: false
+          description: Node / term for which you need to retrieve the range.
+          example: MolecularEntity
```

Comment:
Can we change the example here to `FamilyHistory`? Setting it to `MolecularEntity` does not return anything, and I think `FamilyHistory` is probably easier for user to understand the purpose of this endpoint.